src/MobileUI/Controls/CarouselItem.xaml.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Windows.Input;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Maui.BindableProperty.Generator.Core;
+
+namespace SSW.Rewards.Mobile.Controls;
+
+public partial class CarouselItem : Border
+{
+    public string CarouselImage
+    {
+        get => (string)GetValue(CarouselImageProperty);
+        set => SetValue(CarouselImageProperty, value);
+    }
+
+    public static readonly BindableProperty CarouselImageProperty =
```

Comment:
FYI we could probably use this to make it easier: https://github.com/rrmanzano/maui-bindableproperty-generator

I've lodged this issue for it: https://github.com/SSWConsulting/SSW.Rewards.Mobile/issues/744

Reply:
Oh nice, I was thinking there had to be a better way to go about this... This would be way cleaner.